database/factories/ProductFactory.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Product>
+ */
+class ProductFactory extends Factory
+{
+    protected static $productTypes = ['Laptop', 'Smartphone', 'Tablet', 'Headphones', 'Monitor', 'Keyboard', 'Mouse', 'Printer', 'Camera', 'Speaker','Watch','Modem','AirPurifier', 'Router', 'Projector', 'Scanner', 'Smartwatch', 'Fitness Tracker'];
+    protected static $brand = ['TechPro', 'ElectroMax', 'GadgetZone', 'SmartTech', 'InnoWare', 'FutureTech', 'NextGen', 'PrimeTech', 'EliteGear', 'TechNova', 'TechPlus', 'TechPro', 'TechMax', 'TechUltra', 'TechLite', 'TechBasic'];
+    protected static $type = ['Pro', 'Lite', 'Max', 'Ultra', 'Basic', 'Super', 'Plus', 'Turbo', 'Extreme', 'Elite', 'Prime', 'Ultra', 'UltraMax', 'UltraPro', 'UltraLite', 'UltraBasic'];
+
```

Comment:
Could have used Faker to generate some random words, but felt this made it feel easier to test search functionality later on.